kuma/users/tests/__init__.py

```diff
@@ -27,8 +27,7 @@ def user(save=False, **kwargs):
     password = kwargs.pop('password', 'password')
     user = User(**kwargs)
     user.set_password(password)
-    if save:
-        user.save()
+    user.save()
```

Comment:
Why this change? Not saving seems like a good way to avoid a db write if you don't need it. But if you want to always save we should remove the unused `safe=False` variable as well?